### DISCUSSION TITLE
Improve the bridge and tunnel building mechanics

### DISCUSSION THREAD

**Discussion by gametimeuk** [voxeltycoon/issues#364] [General]
well as spending a bit of time with games that are very similar to this game they allow train tracks across the roads which would be very useful and happens in real life so i do wonder why they havent been included in the game yet

also make a better tunnel and bridge construction as currently its very frustrating to build tunnels and also bridges to the point i end up giving up trying to build them

**Comment by victorleblais** [comment]
This is planned and on the roadmap: https://voxeltycoon.xyz/roadmap

Please check it before posting !

There is already an issue for railway crossingsso you can post there if you want to add details on how these things can work: https://github.com/voxeltycoon/suggestions/issues/26

Although, there are no issue regarding the mechanics of building them. So I'm going to rename this issue to "improve the bridge and tunnel building mechanics".
You are welcome to explain what you don't like or find frustrating about the current way of building those.
And since just saying "it's not good / it's frustrating / it should be better" isn't very constructive, please explain how you think it should work, if you can think of a better way to handle it. Be as precise as you can !

Constructive feedback is always appreciated !
